pkg/controller/kieapp/defaults/auth_test.go

```diff
@@ -595,37 +596,42 @@ func testLDAPLoginModuleRequiredFlag(t *testing.T) {
 			Environment: "rhpam-trial",
 			Objects: api.KieAppObjects{
 				Servers: []api.KieServerSet{
-					{Deployments: Pint(2)},
+					{Deployments: Pint(1)},
 				},
 			},
 			Auth: &api.KieAppAuthObject{
 				LDAP: &api.LDAPAuthConfig{
+					URL:         "ldaps://ldap.example.com",
+					BindDN:      "cn=admin,dc=example,dc=com",
 					LoginModule: "required",
 				},
 			},
 		},
 	}
 	env, err := GetEnvironment(cr, test.MockService())
+	fmt.Println(err)
```

Comment:
it will thrown a null pointer if err is nil, can be removed.

Reply:
Ah yes, I forgot to remove this, added for debug purposes :p